src/OpenTelemetry.Exporter.InMemory/InMemoryMetricExporter.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="InMemoryMetricExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+
+using OpenTelemetry.Metrics;
+
+namespace OpenTelemetry.Exporter
+{
+    public class InMemoryMetricExporter : InMemoryExporter<Metric>
```

Comment:
I think we should mark this class `internal`.

Reply:
That way we don't increase the public API surface. The users just have to use the `public` extension method to add `InMemoryExporter` for Metrics.

Reply:
of course! i'll make the change